### DISCUSSION TITLE
List of Supported sites ?

### DISCUSSION THREAD

**Discussion by frkmusic** [jely2002/youtube-dl-gui#275] [Q&A] [answered]
kindly provide the list of sites , if it allows more than youtube

**Comment by MetallicDoritos** [comment]
Sorry, I didn't see that you had already created a discussion. Here's a small handful of sites that I've successfully used with OVD!
#https://github.com/jely2002/youtube-dl-gui/discussions/524

**Comment by jely2002** [comment, marked as answer]
The full list of supported sites can be found here: https://github.com/yt-dlp/yt-dlp/blob/master/supportedsites.md
